### PR TITLE
Update runtime to 43 and Oomox to 1.15

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

**Comment by actionless** [comment]
thanks! 💯

File: com.github.themix_project.Oomox.json [hunk 1]
```diff
@@ -195,7 +196,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/themix-project/oomox.git",
```
Comment (actionless) [inline com.github.themix_project.Oomox.json:198]: btw URL is still being redirected but it actually changed to `themix-gui` in repo name and i guess in the future Flatpak package-name also will need to follow same change

Comment (bbhtt) [reply com.github.themix_project.Oomox.json:198]: > future Flatpak package-name also will need to follow same change

Do you intend to change the flatpak id to reflect the name change?

Then this one needs to be EOL'd and a new package needs to be submitted.

Comment (actionless) [reply com.github.themix_project.Oomox.json:198]: not sure about that - mb will leave it just as is then for flatpak, to avoid extra steps

**Pull request merged**